### ISSUE TITLE
Parallel Layer Output

### ISSUE THREAD

**Issue by oaishi** [pytorch/captum#383]
Hello, I have four parallel layers just before the final softmax-s of my model. In other words, my model gives four final output. Is it possible to interpret the relation between each individual output and input using captum?

**Comment by bilalsal** [comment]
Hi Oaishi, yes that should be possible.

Have a look at Layer Attribution algorithms in the API reference https://captum.ai/api/layer.html or in the release note https://github.com/pytorch/captum/releases

Hope this helps

**Comment by oaishi** [comment]
Thank you so much for your kind reply!
